src/trace-plugin-loader.js

```diff
@@ -197,19 +147,6 @@ function activate(agent) {
 }
 
 function deactivate() {
-  for (var moduleName in plugins) {
```

Comment:
I suppose since loading plugins is technically a "new" feature this should not have been included to begin with, which is why I removed it here. Also, other changes in this PR change how plugins are stored, and I felt that instead of modifying `deactivate` to reflect this change, I might as well remove it altogether if it will be removed eventually.